examples/oauth2_consumer_webserver/app.py

```diff
@@ -71,7 +71,7 @@ def login():
 def authorized(resp):
     if "error" in request.args:
         return ("<h1>Error</h1>" "<p>{}</p>" '<p><a href="{}">Try again</a></p>').format(
-            request.args["error"], url_for("login")
+            escape(request.args["error"]), url_for("login")
         )
```

Comment:
looks good!

a ~slightly safer approach is to use `markupsafe` to handle the formatting:

```python
return markupsafe.Markup('raw html literal here').format(untrusted, variables, here)
```

Reply:
Ah thanks! I was trying to avoid introducing additional dependencies, but looking at the examples, they could use some updating. 😄 

I'll spend some time next week polishing up these examples and bring in `markupsafe`.

Reply:
flask depends on `markupsafe` via `jinja2` already 👍